sklearnex/preview/ensemble/extra_trees.py

```diff
@@ -1148,7 +1152,6 @@ def _onedal_fit(self, X, y, sample_weight=None, queue=None):
         self.n_features_in_ = X.shape[1]
         if not sklearn_check_version('1.0'):
             self.n_features_ = self.n_features_in_
-        rs_ = check_random_state(self.random_state)
```

Comment:
Seeing this here I would expect you want to add a `check_random_state(self.random_state)` above all the `if` statements.

Reply:
Removing this here was necessary as it is now handled in onedal/ensemble/forest.py. Otherwise we do it twice.